templates/include/telemetry/syncPayload.1.schema.json

```diff
@@ -20,7 +20,9 @@
         "properties": {
           "id": { "type": "string", "pattern": "^[0-9a-f]{64}$" },
           "os": { "type": ["string", "null"] },
-          "version": { "type": "string" }
+          "version": { "type": "string" },
+          "type": { "type": "string" },
+          "syncID": { "type": "string" }
```

Comment:
Note that I'm not updating the include for `syncItem` where the old `devices` field lives. These new type and syncID fields should only ever be populated for clients using the new location directly under payload.